services/121-service/src/metrics/metrics.controller.ts

```diff
@@ -50,7 +50,7 @@ export class MetricsController {
     permissions: [PermissionEnum.RegistrationPersonalEXPORT],
   })
   @ApiOperation({
-    summary: `[SCOPED] Retrieve data for export. Filters only work for export type ${ExportType.allRegistrations}. If no "select" is provided, then the default export columns will be returned.`,
+    summary: `[SCOPED] Retrieve data for export. Filters only work for export type ${ExportType.registrations}. If no "select" is provided, then the hardcoded default select for exporting registrations will be returned.`,
```

Comment:
This is the new behaviour according to my research. @RubenGeo it will use the function `getDefaultSelectForExportRegistrations` whenever a select is not provided. This seems to be intended behaviour but I'm not sure.